linera-core/src/chain_worker/state/mod.rs

```diff
@@ -204,6 +204,7 @@ where
         &mut self,
         proposal: BlockProposal,
     ) -> Result<(ChainInfoResponse, NetworkActions), WorkerError> {
+        tracing::debug!("handling block proposal: {proposal:#?}");
```

Comment:
leftover? We could certainly add TRACE-level log for all handlers.

Reply:
I think this should be done with `tracing::instrument` though.